pkg/ccl/crosscluster/logical/lww_row_processor_test.go

```diff
@@ -420,6 +418,54 @@ func TestLWWConflictResolution(t *testing.T) {
 				}
 				runner.CheckQueryResults(t, fmt.Sprintf("SELECT * from %s", tableNameDst), expectedRows)
 			})
+			t.Run("cross-cluster-local-delete", func(t *testing.T) {
+				if !useKVProc {
+					skip.IgnoreLint(t, "local delete ordering is not handled correctly by the SQL processor")
+				}
+				tableNameDst, rp, encoder := setup(t, useKVProc)
+
+				runner.Exec(t, fmt.Sprintf("INSERT INTO %s VALUES ($1, $2)", tableNameDst), row1...)
+				runner.Exec(t, fmt.Sprintf("DELETE FROM %s WHERE pk = $1", tableNameDst), row1[0])
+
+				// An insert older than the delete is ignored.
+				keyValue2 := encoder(timeOneDayBackward, row2...)
+				require.NoError(t, insertRow(rp, keyValue2, roachpb.Value{}))
+
+				expectedRows := [][]string{}
+				runner.CheckQueryResults(t, fmt.Sprintf("SELECT * from %s", tableNameDst), expectedRows)
+
+				// An insert newer than the delete is respected
+				keyValue3 := encoder(timeOneDayForward, row2...)
+				require.NoError(t, insertRow(rp, keyValue3, keyValue2.Value))
+
+				expectedRows = [][]string{
+					{"1", "row2"},
+				}
+				runner.CheckQueryResults(t, fmt.Sprintf("SELECT * from %s", tableNameDst), expectedRows)
```

Comment:
one more case we could add here:
- remote delete after local delete (to make sure nothing explodes)